src/hockeypuck/server/settings.go

```diff
@@ -304,3 +327,11 @@ func readEnv() map[string]string {
 	}
 	return env
 }
+
+// configureDataDirPaths sets up data directory-based paths for various components
+func (s *Settings) configureDataDirPaths() {
```

Comment:
This is interesting, should we do the same for other paths such as leveldb, stats etc. for consistency?